app/services/validation/date.ts

```diff
@@ -52,3 +120,58 @@ export const createDateSchema = (args?: {
       }
     });
 };
+export const createDateSplitSchema = (args?: {
```

Comment:
nit: could we rename to 
```suggestion
export const createSplitDateSchema = (args?: {
```